source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -288,13 +288,23 @@ Wallets
 |bw|
 -------------------------
 
-    April 4, 2024
+    June 20, 2024
 
-    Version 1.5.1 fixes a bug related to the injected script being loaded multiple times which was causing performance issues on the browser.
+    Version 1.5.2 includes several changes and bug fixes.
+        -   Added new option to edit account name. Name saved in local storage. Changed name displayed across all BrowserWallet.
+        -   Additional error message. Now instead of not showing invalid tokens, they displayed in token list with corresponding error. In order to show, that we found tokens in contract, but they have error.
```

Comment:
I suggest starting with which part/flow of the browser wallet was improved so readers have an idea to which error messages you refer to.

E.g.
```
Improve error messages when searching for all tokenIDs in a given smart contract. Invalid tokenIDs in the contract are now displayed with their corresponding reason for being invalid. 
```